digitaltwin_client/samples/readme.md

```diff
@@ -4,24 +4,98 @@ This directory contains samples demonstrating creating and using Digital Twin in
-* [digitaltwin_sample_device_info](./digitaltwin_sample_device_info) - library that implements a sample device info Digital Twin interface.  This interface reports information about the device - such as OS version, amount of storage, , etc.
-* [digitaltwin_sample_environmental_sensor](./digitaltwin_sample_environmental_sensor) library that implements a sample environmental sensor Digital Twin interface.  This interface demonstrates all concepts of the implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
+### Directories that build executables
+* [digitaltwin\_sample\_device](./digitaltwin_sample_device) - This directory builds the sample executable.  This executable interacts with the Digital Twin interfaces of the neighboring libraries.  It demonstrates creation of interfaces and basic operation when using Azure IoT Hub. It is appropriate for non-resource-constrained devices.
+* [digitaltwin\_sample\_ll_device](./digitaltwin_sample_ll_device) - This directory builds the sample_ll executable. This executable interacts with the Digital Twin interfaces of the neighboring libraries.  It demonstrates creation of interfaces and basic operation when using Azure IoT Hub or Azure IoT Central.  It uses the lower level (LL) layer and enables the Azure IoT Hub Device Provisioning Service (DPS) if using Azure IoT Hub.  It is appropriate for devices with limited resources or single threaded applications and devices.
+
+### Directories that build libraries
+* [digitaltwin\_sample\_device_info](./digitaltwin_sample_device_info) - This directory builds the sample device info library.  This is a Digital Twin interface of helper functions that report information about the device - such as OS version, amount of storage, etc.
+* [digitaltwin\_sample\_environmental_sensor](./digitaltwin_sample_environmental_sensor) - This directory builds the samples environmental sensor library.  This is a Digital Twin interface of helper functions that demonstrates all concepts of implementing a Digital Twin model: commands (synchronous and asynchronous), properties, and telemetry.
 
 ## Getting started
 
-* [Setup to build the Azure IoT C SDK and Digital Twin generally](../doc/building_sdk.md).
+* Build the Digital Twin Client SDK, if not already completed. See [here](../doc/building_sdk.md)
+
+### digitaltwin\_sample\_device
+
+* Use CMake to build the [digitaltwin\_sample\_device](digitaltwin_sample_device) directory.
+
+  From the cmake directory:
+  ```
+  cmake --build .
+  ```
+  This will build the `digitaltwin_sample_device` executable along with the aforementioned sample interfaces.
+
+* Copy the Connection String of the Azure IoT Hub device you wish to enable for Digital Twin:
+
+* Run the sample.
+
+  From the cmake directory:
+  
+  **If using Windows:** 
+
+  ```
+  cd .\digitaltwin_client\samples\digitaltwin_sample_device\Debug\
+  ```
+  Run the executable using your connection string, e.g.:
```

Comment:
Nit: I think you can get rid of the "run the executable" live here and a few lines below.  In the sample_ll_device below you have both the cd and execution step inside the same ``` block and I think it's easier to read like that.  Users just need to copy/paste, making appropriate changes if needed, and don't need to read interspersed text with additional commentary in this case.